docs/topics/proc_starting-your-application-on-openshift-in-debugging-mode.adoc

```diff
@@ -18,11 +32,11 @@ To debug your {runtime}-based application on OpenShift remotely, you must set th
 $ oc get dc
 ----
 
-. Set the `JAVA_DEBUG` environment variable in the deployment configuration of your application to `true`, which configures the JVM to open the port number `5005` for debugging. For example:
+. Set the `{debug-var}` environment variable in the deployment configuration of your application to `true`, which configures the JVM to open the port number `5005` for debugging. For example:
```

Comment:
It'd make sense to document changing the port number here as well via `JAVA_DEBUG_PORT`…

Reply:
I'd just document `JAVA_DEBUG_PORT` on a single place, not two.